src/components/interface/Transactions.vue

```diff
@@ -264,17 +274,15 @@ export default defineComponent({
 
     function sumTransfers(transfers: Transfer[]): string {
       let sum = new BigNumber(0);
-
-      for (let i in transfers) {
-        if (transfers[i].amount?.isGreaterThan(new BigNumber(0))) sum = sum.plus(transfers[i].amount ?? new BigNumber(0));
+      for (const transfer of transfers) {
+        if(transfer.amount > 0) sum = sum.plus(transfer.amount);
```

Comment:
`transfer.amount` is of type `string` when it should be of type `BigNumber` which is why it's throwing a `transfer.amount?.isGreaterThan is not a function` error when using bignumber.js's api

Reply:
Made recommended change to access BigNumber API functions:

`for (const transfer of transfers) {
        const amount = new BigNumber(transfer.amount ?? 0);
        if(amount.isGreaterThan(0)) sum = sum.plus(amount);
  }`